ticdc/manage-ticdc.md

```diff
@@ -807,3 +813,18 @@ enable-old-value = true
 ```
 
 After this feature is enabled, you can see [TiCDC Open Protocol - Row Changed Event](/ticdc/ticdc-open-protocol.md#row-changed-event) for the detailed output format. The new TiDB v4.0 collation framework will also be supported when you use the MySQL sink.
+
+## Replicate the table without a valid index
+
+Since v4.0.8, TiCDC supports replicating the table without a valid index by modifying the task configuration. To enable this feature, configure in the `changefeed` configuration file as follows:
```

Comment:
```suggestion
Since v4.0.8, TiCDC supports replicating tables that have no valid index by modifying the task configuration. To enable this feature, configure in the `changefeed` configuration file as follows:
```